clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +247,105 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStatistics(String cacheAlias, PageSource pageSource) {
```

Comment:
register__Store__Statistics instead and should take a ServerStoreImpl parameter, not a page source